.github/workflows/ci.yml

```diff
@@ -20,23 +16,17 @@ jobs:
       - uses: ./.github/actions/dependencies
       - name: Run Black Test
         run: poetry run black . --line-length=79 --check
-
-
   pycodestyle_test:
     runs-on: ubuntu-latest
     steps:
       - uses : actions/checkout@v3
       - uses: ./.github/actions/dependencies
       - name: Run Pycodestyle Test
         run: poetry run pycodestyle . --ignore=E501
-
-
   pyright_test:
     runs-on: ubuntu-latest
     steps:
       - uses : actions/checkout@v3
       - uses: ./.github/actions/dependencies
       - name: Run Pyright Test
-        run: poetry run pyright
-  
-
+        run: poetry run python -m pyright .
```

Comment:
why do we run this one through python and not the others? does it matter if we run it through python or not?

Reply:
oh no, changed because I was trying things, it doesn't make any difference

Reply:
just make it so that they are all the same; doesn't matter which way you choose.